open-metadata-resources/open-metadata-deployment/ansible-egeria/README.md

```diff
@@ -0,0 +1,166 @@
+
+### Playbooks
+
+Two main playbooks grouping logical sequence of tasks form the related roles.
+
+- `sites.yml` - deploy OMAG server application on site and configure it via admin-service.
+
+- `sites_cleanup.yml` - un-deploy/clean-up OMAG server application form site.
+
+Everything is configurable via set of variables representing one logical application group (i.e. OMAG server profile). For more advanced deployment configurations host groups can be used to configure/deploy multiple instances of OMAG server applications in integrated mode. See configuration and variables section or examples for more details below.
+
+>PRIVILEGE ESCALATION: 
+>With the current version of the playbooks it is assumed that the ansible user can do privilege escalation / sudo to manage system level changes like creating systemd configuration files and controlling (start/stop/restart) the service units. For long term the playbooks will be improved so this will become optional so the system changes can be done beforehand by system admins if needed.
+
+## Usage
+
+#### Deploy
+
+``` ansible-playbook site.yml -i inventories/sample -e target=omagserver1``` 
+
+
+This will trigger sequence of tasks to install, configure and start OMAG server instances on the target groups described as `omagserver1` under inventory.
+
+There is also an option to separately execute sequences by adding following self descriptive tags: `install`,`configure`,`start`.
+
+``` ansible-playbook site_cleanup.yml -i inventories/sample -e target=omagserver1 -t install,configure```
+
+This will install and configure the OMAG application but it will not start it.
+
+#### Un-deploy
+
+``` ansible-playbook site_cleanup.yml -i inventories/sample -e target=omagserver1```
+
+This will invoke sequence of tasks and remove all the files, logs and service related to the specific server application instance on the target systems described in the inventory as `omagserver1`.
+
+>Note that the target is just another input variable mapped to hosts so you can also assign multiple targets as comma separated list like `omagserver1`,`omagserver2` or use the predefined group `all`.
+
+## Roles
+
+### egeria-omag-server
+
+**Deployment** is handled by `egeria-omag-server` role. This is configurable role that will download all the artifacts needed to configure and start egeria OMAG server application. Once this role has been executed against target, specific version of the core Egeria application `server-chassis-spring.jar` will be installed on the pre-configured location. If configured additional Egeria plug-in libraries will be also installed and loaded. One example is `egeria-connector-ibm-information-server-package.jar` if available in the configuration will be downloaded under `libs` folder in the home install directory. System service with pre-configured application name will be also configured and started for you. See role specific configuration and variable section for more details.
```

Comment:
we build a distribution and expect to run in that format. Specifically we place additional libs in server/lib. That may change, and also we place .p12 files in the assembly. For the docker image we extract the assembly an used, so just a heads up this will need maint updates as packaging changes (which may well happen soon)